resources/dex/templates/dex-config-map.yaml

```diff
@@ -12,7 +12,7 @@ data:
     issuer: https://dex.{{ .Values.global.domainName }}
     # extending it to 15days because apparently Istio does not handle ID token validation properly
     expiry:
-      signingKeys: "360h"
+      signingKeys: "15m"
     storage:
```

Comment:
this is already covered in https://github.com/kyma-project/kyma/pull/2736
where btw keys should expire after 1 month :)

Reply:
Yes, this is for testing purposes only. Will be removed before merge.